### PR TITLE
Views created with their respective routes

### LINKED ISSUE #10: Views created with their respective routes / Changes suggested in the CR were made.
# REQUIREMENTS
Move the HTML/CSS views to the Rails project and assign their routes in the 'routes.rb' file. Make the changes suggested in the CR comments.

# DESIGN
image
image
image
image

### PR DESCRIPTION
**Pull request by YxMARCKxY**
# Description
We pass the views to the format required by rails and we relate each view with its respective path.

In addition to making use of the helpers in rails such as:
    -image_tag
    -stylesheet_link_tag
    -javascript_pack_tag
    -link_to

### CODE REVIEW

**Review by LuisGomez89 — DISMISSED** [review]

**Review by YilKanda — CHANGES_REQUESTED** [review]
Be careful, in this Pull Request commits appear from your [PR #9](https://github.com/BC-MAY-21-ROR/kata-07-reloj-checador-k7-n21-t1/pull/9/commits), you need to separate yourself from those commits, and only be above your main branch.

The commits that correspond only to the solution of the issue to be solved must appear, in this case [#10](https://github.com/BC-MAY-21-ROR/kata-07-reloj-checador-k7-n21-t1/issues/10).

File: time-clock/app/javascript/packs/timer-script.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+window.onload = function () {
```
Comment (EduardoGHdez) [inline time-clock/app/javascript/packs/timer-script.js:1]: It'd be nice if we can refactor this functionality to use Stimulus JS to follow the new Rails conventions

**Review by EduardoGHdez — DISMISSED** [review]
LGTM

File: time-clock/app/assets/stylesheets/layouts.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the layouts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (YilKanda) [inline time-clock/app/assets/stylesheets/layouts.scss]: If you don't occupy it, delete it.

File: time-clock/app/helpers/index_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module IndexHelper
+end
```
Comment (YilKanda) [inline time-clock/app/helpers/index_helper.rb]: If you don't occupy it, delete it.

File: time-clock/app/helpers/layouts_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module LayoutsHelper
+end
```
Comment (YilKanda) [inline time-clock/app/helpers/layouts_helper.rb]: If you don't occupy it, delete it.

File: time-clock/app/views/pages/branches.html.erb [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+            <td>North</td>
+            <td>Bugambilais #132, centro Colima Col.</td>
+            <td><i class="fas fa-pencil-alt"></i> &nbsp; &nbsp;<i class="fas fa-toggle-on"></i></td>
+          </tr>
+          </tr>
+          <tr class="register">
+            <td>North</td>
+            <td>Bugambilais #132, centro Colima Col.</td>
+            <td><i class="fas fa-pencil-alt"></i> &nbsp; &nbsp;<i class="fas fa-toggle-on"></i></td>
+          </tr>
```
Comment (YilKanda) [inline time-clock/app/views/pages/branches.html.erb]: Is this repeated code necessary?

File: time-clock/app/views/pages/employees.html.erb [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+          </tr>
+          <tr class="register">
+            <td>Marcelo</td>
+            <td>Developer</td>
+            <td>26359</td>
+            <td>marck_@hotmail.com</td>
+            <td>312-111-11-55</td>
+            <td>Branch 1</td>
+            <td><i class="fas fa-pencil-alt"></i> &nbsp; &nbsp;<i class="fas fa-toggle-on"></i></td>
+          </tr>
```
Comment (YilKanda) [inline time-clock/app/views/pages/employees.html.erb]: Is this repeated code necessary?

File: time-clock/config/database.yml [hunk 7]
```diff
@@ -22,4 +22,6 @@ test:
 
 production:
   <<: *default
-  database: db/production.sqlite3
+  adapter: postgresql
+  database: database_production
+  timeout: 5000
```
Comment (YilKanda) [inline time-clock/config/database.yml:27]: It is not recommended to leave this file on GitHub, I recommend you place it in the .gitignore, and instead replace it with a database.yml.example file.

File: time-clock/Gemfile.lock [hunk 8]
```diff
@@ -230,6 +230,7 @@ DEPENDENCIES
   selenium-webdriver
   spring
   sqlite3 (~> 1.4)
+  stimulus-rails
```
Comment (EduardoGHdez) [inline time-clock/Gemfile.lock:233]: I think you need to run `bundle` again, so the lock gets updated and this dependency gets removed 👍🏽

File: time-clock/.gitignore [hunk 9]
```diff
@@ -5,7 +5,7 @@
 #   git config --global core.excludesfile '~/.gitignore_global'
 
 #Ignore database config
-config/database.yml
+./config/database.yml
```
Comment (EduardoGHdez) [inline time-clock/.gitignore:8]: Is this actually working?
I mean, I just made some changes to this file, and adding the dot won't make anything I think

**Review by EduardoGHdez — APPROVED** [review]
Small comments, none of them should be considered as a blocker 👍🏽 

LGTM

**Pull request merged**